qh_stringinthash.c

```diff
@@ -521,15 +529,19 @@ PHP_METHOD(QuickHashStringIntHash, saveToString)
 	char              *string;
 	uint32_t           string_len;
 
-	php_set_error_handling(EH_THROW, NULL TSRMLS_CC);
```

Comment:
I think zend_replace_error_handling also works in PHP 5.4 and up, so I am okay with just replacing that (if it still works with 5.4 that is).